README.rst

```diff
@@ -177,7 +177,7 @@ Where can I get the latest news about Susi AI?
 -------------------------------------------
 
 Hey, this is the tool for that! Just put
-http://loklak.org/api/search.rss?q=%23susi into your rss reader. Oh
+http://loklak.org/api/search.rss?q=%23susi into your RSS reader. Oh
```

Comment:
Remove the space before `Oh` too.

Reply:
I think space is required over there because of the full stop.